_redirects

```diff
@@ -67,3 +67,5 @@
 /extractors/stripe.html /extractors/tap-stripe
 /extractors/adwords.html /extractors/tap-adwords
 /extractors/salesforce.html /extractors/tap-salesforce
+/singer/maintainers/bytecodeio /
+/extractors/exacttarget.html /
```

Comment:
@lauraperezco do we have an issue around a generic "connector not found page"?  It'd be good to redirect to that eventually rather than just the homepage.

Reply:
@tayloramurphy we don't, but I agree, that'd be better definitely. I can create a separate issue for it. Would that be something for Alex?

Reply:
@lauraperezco I'll add it to the design list for our Designer. It'd be on Alex to implement it, but we should give him a design to work with.